### PR TITLE
Update control-related classes to use Qt versions of data structures.

### PR DESCRIPTION
**Pull request by kberryUSGS**
I updated various Control-related classes to use Qt rather than std versions of data structures. 

I also did a little bit of cleanup and removed `using namespace std;` from files where the std namespace isn't actually used.

See: https://isis.astrogeology.usgs.gov/fixit/issues/5259

### CODE REVIEW

File: isis/src/control/objs/ControlNet/ControlNet.cpp [hunk 1]
```diff
@@ -1925,7 +1925,7 @@ namespace Isis {
    * @returns the radii of the target body
    */
   std::vector<Distance> ControlNet::GetTargetRadii() {
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNet/ControlNet.cpp]: I deliberately didn't change this one to a QVector since it is part of the API.

**Comment by kberryUSGS** [comment]
[Actually PR the QT change into dev.](https://trello.com/c/yUC4UuGu/183-actually-pr-the-qt-change-into-dev)

File: isis/src/control/objs/ControlNetStatistics/ControlNetStatistics.cpp [hunk 2]
```diff
@@ -335,25 +332,25 @@ namespace Isis {
...
-          ostm << "0, 0, 0, 0, 0, 0, 0" << endl;
-        }
+      ostm << mSerialNumList.fileName(it.key()) << ", " << it.key() << ", ";
+      bool serialNumExists = it.value() ;
+      if (serialNumExists) {
+        QVector<double> imgStats = mImageMap[(it).key()] ;
+        ostm << imgStats[imgTotalPoints]<< ", " << imgStats[imgIgnoredPoints] << ", " ;
+        ostm << imgStats[imgLockedPoints] << ", " << imgStats[imgFixedPoints] << ", " ;
+        ostm << imgStats[imgConstrainedPoints] << ", " << imgStats[imgFreePoints] << ", ";
+        ostm << imgStats[ imgConvexHullRatio] << endl;
```
Comment (AgoinsUSGS) [inline isis/src/control/objs/ControlNetStatistics/ControlNetStatistics.cpp]: Looks like there is an unnecessary space here.

File: isis/src/control/objs/ControlNetStatistics/ControlNetStatistics.cpp [hunk 3]
```diff
@@ -335,25 +332,25 @@ namespace Isis {
...
-          ostm << imgStats[imgTotalPoints]<< ", " << imgStats[imgIgnoredPoints] << ", " ;
-          ostm << imgStats[imgLockedPoints] << ", " << imgStats[imgFixedPoints] << ", " ;
-          ostm << imgStats[imgConstrainedPoints] << ", " << imgStats[imgFreePoints] << ", ";
-          ostm << imgStats[ imgConvexHullRatio] << endl;
-        }
-        else {
-          ostm << "0, 0, 0, 0, 0, 0, 0" << endl;
-        }
+      ostm << mSerialNumList.fileName(it.key()) << ", " << it.key() << ", ";
+      bool serialNumExists = it.value() ;
```
Comment (AgoinsUSGS) [inline isis/src/control/objs/ControlNetStatistics/ControlNetStatistics.cpp]: unnecessary spacing before the semicolon

**Comment by kberryUSGS** [comment]
@AgoinsUSGS Thanks for catching those spacing issues! :+1:

**Pull request merged**